build-deploy-platform-services/Jenkinsfile

```diff
@@ -204,20 +200,20 @@ node {
 			echo "NPM Build completed"
 
 			//Change lambda service name to add stack prefix name
-			sh "sed -i -- 's/" + service_name + "/" + var_env_name_prefix + "-" + service_name + "/g' ./deployment-env.yml"
+			sh "sed -i -- 's/" + service_name + "/" + instance_prefix + "-" + service_name + "/g' ./deployment-env.yml"
 
 			echo "Deploying lambda service"
 
-			def envBucketKey = configLoader.JAZZ_S3.jazz_bucket_name_prefix + envmnt
+			def envBucketKey = envmnt.toUpperCase() +  configLoader.JAZZ.S3_BUCKET_NAME_SUFFIX
```

Comment:
is this correct? We seem to have reversed the naming convention. 
Also did we rename the keys from prefix or suffix because it was incorrect to start with or have the values for the keys different now?

Reply:
We had a discussion on this.